src/pages/trade/ui/summary.tsx

```diff
@@ -95,7 +95,7 @@ export const Summary = () => {
       </SummaryCard>
       <SummaryCard title='24h Volume' loading={isLoading}>
         {data && 'directVolume' in data ? (
-          <Density slim>
+          <Density compact>
```

Comment:
issue: this should be `slim` as the token icon gets too big if not, can you revert back?

Reply:
This was causing a lint error for me, might've been on my side. Will fix